src/lib_c/x86_64-windows-msvc/c/sysinfoapi.cr

```diff
@@ -0,0 +1,45 @@
+require "c/winnt"
+require "c/win_def"
+require "c/int_safe"
+
+lib LibC
+  fun GetNativeSystemInfo = GetNativeSystemInfo(system_info : SYSTEM_INFO*)
+
+  struct PROCESSOR_INFO
+    wProcessorArchitecture : WORD
+    wReserved : WORD
+  end
+
+  union OEM_PROCESSOR_INFO
+    dwOemId : DWORD
+    processorInfo : PROCESSOR_INFO
+  end
+
+  struct SYSTEM_INFO
+    oemProcessorInfo : OEM_PROCESSOR_INFO
+    dwPageSize : DWORD
+    lpMinimumApplicationAddress : Void*
+    lpMaximumApplicationAddress : Void*
+    dwActiveProcessorMask : DWORD*
+    dwNumberOfProcessors : DWORD
+    dwProcessorType : DWORD
+    dwAllocationGranularity : DWORD
+    wProcessorLevel : WORD
+    wProcessorRevision : WORD
+  end
+
+  fun GetComputerNameExW = GetComputerNameExW(computer_name_format : COMPUTER_NAME_FORMAT,
```

Comment:
`GetComputerNameExW = ` part is redundant.

Reply:
Fixed in 808e56c.

Reply:
Nice! remember about running `crystal tool format` afterwards :)

Reply:
Yup, did it on the commit just before this one.

Reply:
Needs to be done again, CI is failing:
```
Error: formatting './src/lib_c/x86_64-windows-msvc/c/sysinfoapi.cr' produced changes
```

Reply:
Yeah, weird, will fix.